docs/platform/concepts/backup-to-another-region.rst

```diff
@@ -0,0 +1,70 @@
+
+About the backup to another region
+----------------------------------
+
+The backup to another region (BTAR) feature allows existing backup files to be copied from the service's primary backup region to one or more regions different from the primary one. As a disaster recovery feature, BTAR is particularly useful when the service-hosting region is down since it allows forking the service from an additional copy of the backup residing outside the service-hosting region. In this way, BTAR provides an additional level of security for your data in case of an outage or a downtime on the primary region.
+
+BTAR is currently supported for the following services:
+
+* Aiven for PostgreSQL®
+* Aiven for MySQL®
+
+How it works
+------------
+
+By enabling the backup to another region (BTAR) feature, you create an additional backup on top of the default one located in a primary region where your service is hosted. Contrary to the primary backup, the secondary backup is located outside the primary region.
+
+Secondary backups are taken from primary backups, not from the service itself. For this reason, your secondary backup is available only after a primary backup is there. There might be a data replication lag between the primary region and the secondary region.
+
+If you want to have your service restored from a secondary backup, for example, in case of an outage of the primary region, you create a fork of the service using a region where your secondary backup is located.
+
+.. mermaid::
+
+    flowchart LR
+        subgraph Primary_region
+            direction TB
+            subgraph Service_X
+                end
+            subgraph Primary_backups
+                direction LR
+                PB1
+                PB2
+                PB3
+                PBn
+                end
+            end
+        subgraph Secondary_region
+            direction TB
+            subgraph Forked_service_X
+                end
+            subgraph Secondary_backups
+                direction LR
+                SB1
+                SB2
+                SB3
+                SBn
+                end
+            end
+        Service_X -- Default \n backups --> Primary_backups
+        Primary_backups -- Cross-region backups \n if BTAR  enabled --> Secondary_backups
+        Secondary_backups -- Secondary backups \n to restore service X --> Forked_service_X
+        Service_X -- Forking service X \n if primary region down --> Forked_service_X
+
+Limitations
+-----------
+
+* When selecting a cloud region for your additional backup, you need to use the same cloud provider that your service uses.
+* When you want to :ref:`restore your service from an additional backup <fork-and-restore>` and you use a point in time to specify the scope of data to be backed up, you need to set up the time to no later than the time of taking the latest backup.
```

Comment:
```suggestion
* When you want to :ref:`restore your service from an additional backup <fork-and-restore>` and you use a point in time to specify the scope of data to be restored, you need to set up the time to no later than the time of latest backup.
```

this is actually a limitation, that only upto the latest backup can be restored (from secondary location) and nothing after that. This limitation will also be lifted in future